functorch/_src/partitioners.py

```diff
@@ -368,7 +380,10 @@ def get_node_weight(node):
         if ban_recomputation(node) and node in required_fw_nodes:
             nx_graph.add_edge("source", node.name + "_in", capacity=math.inf)
 
-        if 'tensor_meta' not in node.meta:
+        if is_sym_node(node):
+            weight = 1
```

Comment:
Probably a basic question - but what's the idea behind `weight = 1` for sym nodes?

Reply:
weight refers to the memory footprint as viewed from the perspective of cost for saving the value for backward.  the partitioner uses it for min-cut.  weight=1 is literally correct for symint/symfloats, other than i'm not sure if we factor eltsize into the weight of a tensor or just numel.

Reply:
oh ok - put another way, I guess that this is saying that all symint nodes in the graph are trivially cheap to save for the backward (compared to most tensor nodes, which have a memory footprint corresponding to their size)?

Reply:
exactly.  short of actually saying 'lets always save all the symints' (which we don't, if they aren't really used), this informs the partitioner of their relative cheapness to save.